content/hardware/01.mkr/01.boards/mkr-wan-1310/tutorials/lora-message/lora-message.md

```diff
@@ -8,7 +8,7 @@ tags:
  - LoRa®
 author: 'Karl Söderby'
 libraries: 
-  - name: LoRa
+  - name: LoRa®

```

Comment:
Thats the library name, so we should sync with the library's name
```suggestion
  - name: LoRa

```

Reply:
changed